### PR TITLE
Fix builder metadata

### PR DESCRIPTION
**Pull request by waveform80**
Fix some failures in the Builder.metadata property and ldd dependency logic

### CODE REVIEW

File: piwheels/slave/builder.py [hunk 1]
```diff
@@ -211,14 +220,30 @@ def metadata(self):
         """
         if self._metadata is None:
             with zipfile.ZipFile(self.open()) as wheel:
-                filename = (
+                filenames = {
```
Comment (bennuttall) [inline piwheels/slave/builder.py:223]: This is a set containing two values, but it's not obvious that's the case. Each part of the filename structure being on a separate line makes it harder to parse. If each one doesn't fit on a single line I'd probably create two string variables and create a set containing them both (or go with `for filename in {foo, bar}`), although admittedly, naming them would be tricky.

File: tests/test_info.py [hunk 2]
```diff
@@ -151,3 +142,9 @@ def test_get_cpu_temp():
         assert get_cpu_temp() == 76.543
         m.return_value.__enter__.side_effect = FileNotFoundError
         assert get_cpu_temp() == 0.0
+
+
+def test_get_pi_info():
+    assert get_pi_info('a02082') == PiInfo('3B', '1.2', 'BCM2837', 'Sony', '1Gb')
```
Comment (bennuttall) [inline tests/test_info.py:148]: Should memory not be measured in GB? (I've made a similar comment before in gpiozero)

**Review by bennuttall — APPROVED** [review]
Couple of minor comments but generally LGTM

File: tests/test_info.py (hunk 2 above)
Comment (waveform80) [reply tests/test_info.py:148]: Yes :)

**Pull request merged**